README.md

```diff
@@ -69,7 +69,7 @@ To learn more about the docker pull rate limits and the open source software pro
 | [FLASH](https://hub.docker.com/r/staphb/flash) <br/> [![docker pulls](https://badgen.net/docker/pulls/staphb/flash)](https://hub.docker.com/r/staphb/flash) | <ul><li>1.2.11</li></ul> | http://ccb.jhu.edu/software/FLASH |
 | [Flye](https://hub.docker.com/r/staphb/flye) <br/> [![docker pulls](https://badgen.net/docker/pulls/staphb/flye)](https://hub.docker.com/r/staphb/flye) | <ul><li>2.5</li><li>2.7</li><li>2.8</li><li>2.9</li><li>2.9.1</li></ul> | https://github.com/fenderglass/Flye |
 | [Freyja](https://hub.docker.com/r/staphb/freyja) <br/> [![docker pulls](https://badgen.net/docker/pulls/staphb/freyja)](https://hub.docker.com/r/staphb/freyja) | <ul><li>1.2</li><li>1.2.1</li><li>1.3.1</li><li>1.3.2</li><li>1.3.4</li><li>1.3.7</li><li>1.3.8</li><li>1.3.9</li><li>1.3.10</li><li>1.3.11</li></ul> | https://github.com/andersen-lab/Freyja |
-| [GAMBIT](https://hub.docker.com/r/staphb/gambit) <br/> [![docker pulls](https://badgen.net/docker/pulls/staphb/gambit)](https://hub.docker.com/r/staphb/gambit) | <ul><li>0.3.0</li><li>0.4.0</li><li>0.5.0</li></ul> | https://github.com/hesslab-gambit/gambit |
+| [GAMBIT](https://hub.docker.com/r/staphb/gambit) <br/> [![docker pulls](https://badgen.net/docker/pulls/staphb/gambit)](https://hub.docker.com/r/staphb/gambit) | <ul><li>0.3.0</li><li>0.4.0</li><li>0.5.0</li><li>1.0.0</li></ul> | https://github.com/jlumpe/gambit |
```

Comment:
New repo!

I love when tools are actively maintained. I tried to look for 'https://github.com/hesslab-gambit/gambit', but I was directed to 'https://github.com/jlumpe/gambit', so I don't think we need to worry about old links.

Reply:
Same repo, ditched the old organization and just moved it back under my own user. Aside from this and the BioConda recipe I don't think anyone cared about that URL.

Reply:
Could you add your handle at the bottom of this README file?

Reply:
GitHub username?

Reply:
@jlumpe - Yes GitHub username for the list at the bottom of the main `/README.md`. 

It is not required to add, but we like to keep a list of the folks that have contributed and give them credit. Only add if you're comfortable with it